src/GuiBase/Utils/KeyMappingManager.cpp

```diff
@@ -10,7 +10,8 @@ namespace Ra::Gui {
 using namespace Core::Utils; // log
 
 KeyMappingManager::KeyMappingManager() :
-    m_defaultConfigFile( std::string( Core::Resources::getRadiumResourcesDir() ) +
+    ///\todo how to check here ?
+    m_defaultConfigFile( *Core::Resources::getRadiumResourcesPath() +
```

Comment:
do we need to check here ? 
what behavior ? 
should add dependency to RadiumEngine (don't know if it's initialize before btw).

Reply:
from https://en.cppreference.com/w/cpp/utility/optional/operator*, `The behavior is undefined if *this does not contain a value.`.
Concl. Yes, we have to check. No keymapping if config file not found.
I'm quite against adding a dependency to the Engine.